crowbar_framework/config/locales/dns/en.yml

```diff
@@ -20,6 +20,7 @@ en:
     dns:
       edit_attributes:
         domain: 'Domain'
+        domain_hint: 'Changing the domain after deployment is not supported.'
```

Comment:
I would say ***after initial deployment*** (It's probably implied, but kind of looks better for me)